scanner/updater/manual/manual.go

```diff
@@ -1,39 +1,220 @@
-// Package manual provides a custom updater for vulnerability scanner.
+// Package manual provides a custom updater for vulnerability scanners.
 // This updater allows manual input of vulnerability data.
 package manual
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
 
+	"github.com/pkg/errors"
 	"github.com/quay/claircore"
 	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/zlog"
+	"github.com/stackrox/rox/pkg/httputil/proxy"
+	"github.com/stackrox/rox/pkg/utils"
 )
 
-// Name provides the name for the updater.
-const Name = `stackrox-manual`
+type Vulnerability struct {
+	Name               string `json:"Name"`
+	Description        string `json:"Description"`
+	Issued             string `json:"Issued"`
+	Links              string `json:"Links"`
+	Severity           string `json:"Severity"`
+	NormalizedSeverity string `json:"NormalizedSeverity"`
+	Package            struct {
+		Name           string `json:"Name"`
+		RepositoryHint string `json:"RepositoryHint"`
+	} `json:"Package"`
+	IntroducedInVersion string `json:"IntroducedInVersion"`
+	FixedInVersion      string `json:"FixedInVersion"`
+	Repo                struct {
+		Name string `json:"Name"`
+		URI  string `json:"URI"`
+	} `json:"Repo"`
+}
 
-// UpdaterSet creates a new updater set with the provided vulnerability data.
-func UpdaterSet(_ context.Context) (driver.UpdaterSet, error) {
-	res := driver.NewUpdaterSet()
-	err := res.Add(&updater{})
+type Vulnerabilities struct {
+	Vulnerabilities []Vulnerability `json:"vulnerabilities"`
+}
+
+const (
+	Name = "stackrox-manual"
+	// DefaultURL Default URL to fetch the vulnerabilities JSON.
+	DefaultURL = "https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.json"
```

Comment:
Though this is here, I do also think it'd be good to add this as the default to the CLI, so it's clear to users what the default is